Formula/n/no-more-secrets.rb

```diff
@@ -3,7 +3,7 @@ class NoMoreSecrets < Formula
   homepage "https://github.com/bartobri/no-more-secrets"
   url "https://github.com/bartobri/no-more-secrets/archive/refs/tags/v1.0.1.tar.gz"
   sha256 "4422e59bb3cf62bca3c73d1fdae771b83aab686cd044f73fe14b1b9c2af1cb1b"
-  license "GPL-3.0"
+  license "GPL-3.0-only"
```

Comment:
https://github.com/bartobri/no-more-secrets/blob/v1.0.1/nms.6#L39

```suggestion
  license "GPL-3.0-or-later"
```